invenio_rdm_records/resources/serializers/dcat/__init__.py

```diff
@@ -6,21 +6,32 @@
 # it under the terms of the MIT License; see LICENSE file for more details.
 
 """Datacite to DCAT serializer."""
+import mimetypes
 
 from datacite import schema43
+from flask_resources import BaseListSchema, MarshmallowSerializer
+from flask_resources.serializers import SimpleSerializer
 from lxml import etree as ET
 from pkg_resources import resource_stream
 from werkzeug.utils import cached_property
 
-from invenio_rdm_records.resources.serializers import DataCite43XMLSerializer
+from invenio_rdm_records.contrib.journal.processors import JournalDataciteDumper
+from invenio_rdm_records.resources.serializers.dcat.schema import DcatSchema
 
 
-class DCATSerializer(DataCite43XMLSerializer):
+class DCATSerializer(MarshmallowSerializer):
     """DCAT serializer for records."""
 
     def __init__(self, **options):
         """Constructor."""
-        super().__init__(encoder=self._etree_tostring)
+        super().__init__(
```

Comment:
I am not a huge fan of this change...

DCAT is based on DataCite and I feel we're duplicating code just to add `files`. However, adding `files` directly to Datacite's schema does not feel right either. 

Feel free to suggest other alternatives.